pkg/dispatchcli/cmd/cmd.go

```diff
@@ -35,6 +36,7 @@ var validResources = i18n.T(`Valid resource types include:
 	* install
 	* secrets
 	* subscription
+	* uninstall
```

Comment:
I think install and uninstall do not belong here, actually this whole variable could be removed. The idea for this variable was to generalize `get`, but since we build subcommand for every type of resource, `dispatch get --help` prints the list anyway. Seems like this variable is not needed.